hub/util/path.py

```diff
@@ -73,6 +73,8 @@ def get_path_type(path: Optional[str]) -> str:
         return "s3"
     elif path.startswith("gdrive://"):
         return "gdrive"
+    elif path.startswith("https://ipfs.infura") or path.startswith("http://127") or path.startswith("https://api.web3.storage") or path.startswith("https://shuttle-5.estuary"):
+        return "ipfs"
```

Comment:
This will never be reached as "http://" and "https://" are already covered in the if ladder.